### DISCUSSION TITLE
Remote Start Deprecation

### DISCUSSION THREAD

**Discussion by stboch** [G-Two/homeassistant-subaru#51] [General]
While the other services make sense because frankly they are just buttons. I think keeping the remote start service is good because you can control the climate preset more easily via automation then have to alter the select.

Just my opinion though...

**Comment by G-Two** [comment]
That's a good point. I can keep remote start around, but I'll probably modify it so it uses the device id instead of the VIN.

**Reply by stboch** [reply]
yeah true device ID is a good idea.

**Reply by makokiller** [reply]
How do you find the device ID?
